ui/src/shared/components/Gauge.tsx

```diff
@@ -366,7 +367,13 @@ class Gauge extends Component<Props> {
     const {degree, needleColor0, needleColor1} = GAUGE_SPECS
     const arcDistance = Math.PI * 1.5
 
-    const needleRotation = (gaugePosition - minValue) / (maxValue - minValue)
+    const limitedGaugePosition = Math.max(
```

Comment:
This fix keeps the gauge needle at most the min/max values

Reply:
maybe this comment would be useful in the code itself?